source/getting-started/atlas-log-in.txt

```diff
@@ -72,9 +72,9 @@ Steps
       .. tip::
 
          You can also initiate the log in process from the 
-         :guilabel:`Code Generation` tab by clicking the 
-         :icon-lg:`LogIn` :guilabel:`Log in with Atlas` button on the 
-         :guilabel:`Query Converter` pane.
+         :guilabel:`Code Generation` tab. To start the login process, 
```

Comment:
(praise)
This is the right structure. Nice!

(minimalism)
What about just "To log in, ... "

Reply:
The lead in here gives context that there are multiple ways to initiate the login process so it should stay.